src/posts/introducing-pint-xarray/index.md

```diff
@@ -0,0 +1,288 @@
+# Unit-aware arithmetic in xarray, via pint
```

Comment:
> @andersy005, do I need to do something special to be able see a log of the failing build? I get 404s when I try to follow the links.

@keewis, we are missing the necessary metadata information... The following changes should fix the issue 

```suggestion
---
title: "Unit-aware arithmetic in Xarray, via pint"
date: "2022-07-06"
authors:
  - Justus Margin
  - Tom Nicholas
summary: "Xarray now supports unit-aware operations by wrapping pint arrays"
---
```